### ISSUE TITLE
Unknown materials/data entries

### ISSUE THREAD

**Issue by CMDR-Cometborne** [EDDiscovery/EDDiscovery#1110]
The new materials and data added in 2.3.10 are not correctly recognized in the materials panel, see attached screenshot taken in 8.0.9.0:

![image](https://user-images.githubusercontent.com/25147462/27851919-37d966f4-615d-11e7-87a7-e1e58c9f2e8e.png)

The in-game names and information are:
Unknown Structural Data - data, grade 2
Unknown Material Composition Data - data, grade 3
Unknown Carapace - material, grade 5
Unknown Energy Cell - material, grade 3
Unknown Technology Components - material, grade 4
Unknown Residue Data Analysis - data, grade 4
Unknown Organic Circuitry - material, grade 5

**Comment by AnthorNet** [comment]
Are you sure the materials are materials?
I added them to EDSM but still receive them as part of the Data API.

I'll need to check which software sends them like that.
And if you could past your journal, it would be appreciated.

**Comment by CMDR-Cometborne** [comment]
Pretty sure about the material/data breakdown.  In-game screenshot of the materials panel:
![image](https://user-images.githubusercontent.com/25147462/28005844-fa082876-654c-11e7-84c2-6482f71eb291.png)

And the data panel:
![image](https://user-images.githubusercontent.com/25147462/28005872-18ad6ffc-654d-11e7-8d60-2ae4e16669f7.png)

I can give you the journal files, will ping on discord to organize details.

**Comment by klightspeed** [comment]
In the materials window, "Encoded" means it's data, while "Manufactured" means it's materials.

EDDiscovery just needs the name mapping in EDDiscovery/EliteDangerous/MaterialCommodities.cs

* Data: Tg Structuraldata -> Unknown Structural Data
* Data: Tg Compositiondata -> Unknown Material Composition Data
* Data: Tg Residuedata -> Unknown Residue Data Analysis
* Material: Unknowncarapace -> Unknown Carapace
* Material: Unknownenergycell -> Unknown Energy Cell
* Material: Unknowntechnologycomponents -> Unknown Technology Components
* Material: Unknownorganiccircuitry -> Unknown Organic Circuitry

```
AddEnc("Unknown Structural Data", "Common", "USD", "Tg Structuraldata");
AddEnc("Unknown Material Composition Data", "Standard", "UMCD", "Tg Compositiondata");
AddEnc("Unknown Residue Data Analysis", "Rare", "URDA", "Tg Residuedata");
AddManu("Unknown Energy Cell", "Standard", "UEC");
AddManu("Unknown Technology Components", "Rare", "UTC");
AddManu("Unknown Carapace", "Very Rare", "UC");
AddManu("Unknown Organic Circuitry", "Very Rare", "UOC");
```

**Comment by robbyxp1** [comment]
Pushed this fix to 8.1, reviewed ED Data.
